assets/js/ModuleCategory.js

```diff
@@ -76,44 +50,32 @@ export default class ModuleCategory extends React.Component {
     // Grabs icon from first module in category for category icon
     var icon = 'icon-' + this.props.modules[0].props.icon + ' ml-icon';
 
-    var catName = (this.props.libraryOpen)
-
-    ? <div className='cat-container'>
-        <div className={sort} />
-        <span className='open-ML-cat content-3 t-vl-gray'>{this.props.name}</span>
-      </div>
-    : <div className='cat-container closed-ML-cat'>
-        <span className={icon}></span>
-      </div>
-
-    var hideAnimation = (!!this.state.visible) ? 'hide-animation' : null;
+    var categoryHead = (this.props.libraryOpen)
+      ? <div className='first-level' onClick={this.toggleCollapse} >
+          <div className='cat-container' >
+            <div className={sort} />
+            <span className='open-ML-cat'>{this.props.name}</span>
+          </div>
+        </div>
+      : <div className='first-level' onMouseEnter={this.toggleCollapse} >
+          <div className='cat-container closed-ML-cat' >
+            <span className={icon}></span>
+          </div>
+        </div>
 
-    // 'on-*' props trigger an objection from Facebook, see: https://reactjs.org/warnings/unknown-prop.html
-    var collapse = (this.props.libraryOpen)
-      ? <Collapse className='' isOpen={isOpen}>
+    var moduleList = (this.props.libraryOpen)
```

Comment:
Generally I prefer if statements to ?:, except for very short and simple expressions.

Reply:
Noted, changing